GitUI/Editor/FileViewerInternal.cs

```diff
@@ -131,15 +131,12 @@ public void SetText(string text, Action openWithDifftool, bool isDiff = false)
                 _lineNumbersControl.DisplayLineNumFor(text);
             }
 
-            TextEditor.BeginUpdate();
-
             TextEditor.Text = text;
 
             // important to set after the text was changed
             // otherwise the may be rendering artifacts as noted in #5568
             TextEditor.ShowLineNumbers = !isDiff;
-
-            TextEditor.EndUpdate();
+            TextEditor.Refresh();
```

Comment:
IIRC, without Being/EndUpdate the control would badly flicker...
I'm going to test it

Reply:
Here's a flicker example:

![flicker](https://user-images.githubusercontent.com/4403806/49578533-16f36900-f99e-11e8-9f69-f1166438f8c3.gif)

Reply:
How about now? I put the beginupdate back. It still fixes the problem and the flicker is gone.

Reply:
Ignore my previous comment.... with the beginupdate, the but is back...:(

Reply:
Ok, this one helps a little for reducing the flicker! I just put the beginupdate back, and put the refresh outside the update block.